src/test/ui/const-generics/const-arg-type-arg-misordered.stderr

```diff
@@ -14,6 +14,7 @@ LL | fn foo<const N: usize>() -> Array<N, ()> {
    |                                   ^
    |
    = note: type arguments must be provided before constant arguments
+   = help: reorder the arguments: lifetimes, then types, then consts: `<T, N>`
```

Comment:
This should say "types, then consts:". I've noticed this is incorrect with the generic parameter message too, for which I've opened up https://github.com/rust-lang/rust/issues/72816.

Reply:
I'll fix that too in this PR